src/Prism.Core/Commands/DelegateCommand{T}.cs

```diff
@@ -100,7 +111,19 @@ public bool CanExecute(T parameter)
         /// <param name="parameter">Command Parameter</param>
         protected override void Execute(object parameter)
         {
-            Execute((T)parameter);
+            try
+            {
+                // Note: We don't call Execute because we would potentially invoke the Try/Catch twice.
+                // It is also needed here incase (T)parameter throws the exception
```

Comment:
Looks like we need the same try/catch coverage on the (t)parameter in CanExecute on line 136

Reply:
done